### PR TITLE
Broaden

### PR DESCRIPTION
**Pull request by HelenCEBM**
- Include patients treated **in hospital** for MABs according to BlueTeq data, and look for corresponding spells in SUS
- For non-hosp, include all short stay elective rather than just day cases
- Update notebook to handle new data

### CODE REVIEW

File: analysis/study_definition.py [hunk 1]
```diff
@@ -91,48 +91,57 @@
...
-  hospital_admission_date = patients.admitted_to_hospital(
+  ### Any elective admission with COVID as primary diagnosis (with or without a recorded procedure)
+  elective_admission_date = patients.admitted_to_hospital(
     returning = "date_admitted",
     with_these_primary_diagnoses = covid_icd10_codes,
-    with_patient_classification = ["2"], # day case admissions only
+    with_patient_classification = ["1", "2"], # ordinary or day case admissions only
+    with_admission_method = ['11', '12', '13', '81'], ## elective (inc transfer)
     # see https://docs.opensafely.org/study-def-variables/#sus for more info
     between = ["outpatient_covid_therapeutic_date - 3 days", "outpatient_covid_therapeutic_date + 3 days"],
```
Comment (acagreen17) [inline analysis/study_definition.py:146]: What's the rational for 3 days? What did the group who have done this already find the lag was? Is it worth expanding and filtering post extraction?

**Review by acagreen17 — COMMENTED** [review]
Looks good. Made a few small comments but most probably just my misunderstanding of how to extracted sus data rather than any actual issues.

File: analysis/study_definition.py (hunk 1 above)
Comment (HelenCEBM) [reply analysis/study_definition.py:146]: I think they used 2 days and the tails were small so I think 3 should capture all except for the largest data entry errors in the treatment date

File: analysis/study_definition.py [hunk 2]
```diff
@@ -198,30 +244,139 @@
...
 
+
+  ### To match with Blueteq INPATIENT treatment
+  ### Any admission (SUS) (include elective in case of hospital-onset COVID) Without procedure: 
+  any_admission_date = patients.admitted_to_hospital(
+    returning = "date_admitted",
+    with_these_diagnoses = covid_icd10_codes,
+    with_patient_classification = ["1"], # ordinary admissions only
+    # see https://docs.opensafely.org/study-def-variables/#sus for more info
+    on_or_before = "inpatient_covid_therapeutic_date",
```
[the remark this answers is not in the document]
Comment (HelenCEBM) [reply analysis/study_definition.py:268]: I'm using on_or_before here for inpatient treatments to ensure patient was already admitted prior to/on blueteq treatment date (the spell could be very long in some cases) - then filtering later to make sure the treatment date was also on or before the discharge date (don't think this is easily possible in study definition).

File: analysis/study_definition.py [hunk 3]
```diff
@@ -198,30 +244,139 @@
...
+    },
+  ),
+  
+  ### [Discharge date] Any admission (SUS) without procedure: 
+  any_discharge_date = patients.admitted_to_hospital(
+    returning = "date_discharged",
+    with_these_diagnoses = covid_icd10_codes,
+    with_patient_classification = ["1"], # ordinary admissions only
+    # see https://docs.opensafely.org/study-def-variables/#sus for more info
+    between = ["any_admission_date","any_admission_date"],
```
[the remark this answers is not in the document]
Comment (HelenCEBM) [reply analysis/study_definition.py:284]: As above this is for inpatient/in-hospital treatments from Blueteq and includes any type of admission (patient classification = 1 is all except day cases and regular admissions) - this is used later to check that the treatment date fell within the spell

**Pull request merged**